graphene/types/union.py

```diff
@@ -56,7 +56,7 @@ def __init_subclass_with_meta__(cls, types=None, **options):
             isinstance(types, (list, tuple)) and len(types) > 0
         ), f"Must provide types for Union {cls.__name__}."
 
-        _meta = UnionOptions(cls)
+        _meta = options.pop("_meta", UnionOptions(cls))
```

Comment:
Can you please add _meta as an explicit argument, similar to base.py or enum.py? I'd like to keep things consistent 🙂